.github/workflows/backwards_compatibility_tests_workflow.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       matrix:
         java: [ 11, 17, 21 ]
         os: [ubuntu-latest,windows-latest]
-        bwc_version : ["2.9.0","2.10.0","2.11.0","2.12.0","2.13.0","2.14.0"]
+        bwc_version : ["2.9.0","2.10.0","2.11.0","2.12.0","2.13.0","2.14.0","2.15.0","2.16.0-SNAPSHOT"]
```

Comment:
do you know why we're adding two versions after 2.15 release? Does it mean we haven't had bwc for 2.15-snapshot after 2.14 release?

Reply:
yes